macros/ArticlesByTag.ejs

```diff
@@ -16,6 +16,7 @@
 var l10n = {
     noArticles : mdn.localString({
         "en-US": "There have been no articles written yet. <a href=\"/en-US/docs/MDN/Getting_started\">Please consider contributing</a>.",
+        "zh-CN": "这些文章尚未撰写。<a href=\"/en-US/docs/MDN/Getting_started\">欢迎贡献</a>。",
```

Comment:
Translation looks good, but the link should go to [/zh-CN/docs/MDN/Getting_started](https://developer.mozilla.org/zh-CN/docs/MDN/Getting_started). This macro is a good candidate for deletion, it is only used on [9, non-English pages](https://developer.mozilla.org/en-US/search?locale=%2A&kumascript_macros=ArticlesByTag&topic=none).